server/src/test/java/org/elasticsearch/index/store/StoreTests.java

```diff
@@ -815,7 +815,7 @@ public void testStoreSizes() throws IOException {
         // directory that returns total written bytes as the data set size
         final var directory = new ByteSizeDirectory(StoreTests.newDirectory(random())) {
 
-            final AtomicLong dataSetBytes = new AtomicLong(0L);
+            final AtomicLong dataSetBytes = new AtomicLong(estimateSizeInBytes(getDelegate()));
```

Comment:
I am new to this area of code. Please let me know if the fix makes sense. Thanks!